docs/api-proposal/proposed-routes.md

```diff
@@ -0,0 +1,76 @@
+This doc proposes a new set of API endpoints for the multi-jurisdiction audit flow.
+
+Some principles I tried to follow:
+
+- Reuse existing routes when available
+- Use REST conventions
+- Avoid reusing routes across roles. Instead of reusing one route and returning different scoped responses, make different roles query different endpoints with explicit scoping in the route. That way we just have to check if a user has access to the data they are asking for, as opposed to trying to filter data based on their access. (I learned this one the hard way.)
+
+New routes are in **bold**.
+
+# Audit Admin: Audit setup
+
+![](./create-audit.png)
+
+- POST /election/new - create new audit
+- **GET /election** - get all audits
+
+![](upload-jurisdiction-file.png)
+
+- POST /election/<election_id>/jurisdiction/csv - upload CSV file of jurisdictions with JAs
+- GET /election/<election_id>/jurisdiction/csv - download CSV file of jurisdictions with JAs
+
+![](contests.png)
+
+- **PUT /election/<election_id>/contests** - create/update all contests, including contest universe
+- **GET /election/<election_id>/contests** - get all contests
```

Comment:
:+1:

Reply:
This was meant to be singular `/contest` to match the naming convention elsewhere